### ISSUE TITLE
`Error: not a directory`

### ISSUE THREAD

**Issue by brillout** [AurelienLourot/uncommitted#8]
Hi!

In order to avoid hidden files I run uncommitted with a wildcard
```
uncommitted -Luns ~/*
```

But I get

```
Error: not a directory: /home/romu/DSC05907 (1).JPG
```

Could be nice to just ignore files instead of returning an error.

Thanks for the tool!

**Comment by AurelienLourot** [comment]
Meanwhile, I think you could just do

**Comment by brillout** [comment]
Yes I thought about finding a workaround.
Thanks for providing one. And It seems to work.

But the ticket was meant more as a usability issue and less as a feature request per se.

**Comment by AurelienLourot** [comment]
**TL;DR**: it's not a workaround because `*` and `*/` are bash features, not features of `uncommitted`. The issue here is that you're not using the proper bash expression.

**Details**:

Basically when you use `*`, your bash expands it and passes the resulting list to `uncommitted`. So `uncommitted` doesn't know you were passing a wildcard. It could as well be that you've passed a list and made a typo in one item, and in this case you would want `uncommitted` to warn you.

Now I tend to think that this is actually good that `uncommitted` warns you if some elements of your list aren't folders. This gives you a good hint to fix your list, and in your use case that means passing a trailing slash to bash, so it produces the list that you really wanted to pass to `uncommitted`.

**Comment by AurelienLourot** [comment]
In fact, making `uncommitted` silent here would be a workaround for your wrong list @brillout :) I'd prefer not to change this behavior if you don't mind. But thanks to this issue I learned about `*/`, so thanks a lot!

**Comment by brillout** [comment]
Ok yes makes sense:)
